examples/experimentation-framework/clutch-config.yaml

```diff
@@ -25,5 +25,15 @@ modules:
       rtds_layer_name: foo
       cache_refresh_interval: "60s"
 services:
+  - name: clutch.service.db.postgres
```

Comment:
we should probably remove this

Reply:
Are you using this file? If so, are you using a mock server or the real server?

If the purpose of this file to serve as an example, I think it makes sense for it to use a real postgres DB instead of the mock store.

Reply:
Ah, I see - I think that it makes sense to keep it then - leaving the decision up to you. I haven't used this file but I agree - it's good to have some kind of an example for how it should look like.